app/main/helpers/email.py

```diff
@@ -0,0 +1,63 @@
+from flask import url_for, current_app
+import mandrill
+from itsdangerous import URLSafeTimedSerializer
+
+from .. import main
+
+
+with open("./email_templates/forgotten_password_email.html", "r") as f:
+    forgot_password_email_body = f.read()
+
+
+def send_password_email(user_id, email_address):
+    try:
+        mandrill_client = mandrill.Mandrill(main.config['MANDRILL_API_KEY'])
+        url = generate_reset_url(user_id, email_address)
+        body = forgot_password_email_body.format(url)
```

Comment:
I think putting them in a separate subdirectory would be separate enough. If we want to keep them completely separate we could use Jinja2 directly. I think using the same templating language throughout makes template authoring simpler.